lms/envs/common.py

```diff
@@ -396,6 +396,9 @@
 
     # Whether the bulk enrollment view is enabled.
     'ENABLE_BULK_ENROLLMENT_VIEW': False,
+
+    # Whether course goals is enabled.
+    'ENABLE_COURSE_GOALS': True,
```

Comment:
Why do you have a setting and a flag?

Reply:
^ for open source and white label to turn it off globally once waffle flag is removed